src/main/java/org/wso2/carbon/module/swiftiso20022/JsonToMT940Transformer.java

```diff
@@ -56,8 +54,8 @@ public void connect(MessageContext messageContext) throws ConnectException {
             if (payload.isPresent()) {
                 RequestPayloadModel requestPayload = gson.fromJson(payload.get(), RequestPayloadModel.class);
```

Comment:
Can't we do this after the validation check?

Reply:
This will check for all the payload empty null and black scenarios and throw an error. That's why I did those before the attribute validations